### ISSUE TITLE
More built-in options for point images

### ISSUE THREAD

**Issue by bjasnow** [scholarslab/Neatline#179]
It would be very convenient to have an array of map-icons to choose from, so that points indicating places on the map could tell more of a story. For example, numerical items, items shaped like forts or castles, items shaped like temples/churches, bridges, etc.

**Comment by davidmcclure** [comment]
Yeah, this is a great idea. I've thought about doing this by just having an "images" directory in the plugin that comes pre-stocked with a bunch of free icon sets (eg, http://glyphicons.com/). Everything in that directory would get sucked up by the front-end application and made available for selection in some kind of simple interface in the editor.

That way, the user could always just drop new images into that directory, and they'd be instantly available for use via the GUI.
